Existing Tests/Factories are broken

Issue in `core/factories.py`
```
dashboard/tests/tests_dashboard_views.py:4: in <module>
    from core.factories import UserFactory
core/factories.py:7: in <module>
    class UserFactory(DjangoModelFactory):
core/factories.py:12: in UserFactory
    @post_generation
E   NameError: name 'post_generation' is not defined

```

Issue In `UserFactory`
```
class UserFactory(DjangoModelFactory):
    username = factory.Faker("user_name")
    email = factory.Faker("email")
    name = factory.Faker("name")
```
`User` model doesn't have any attribute `name`.

```
FAILED dashboard/tests/tests_dashboard_views.py::QuestionModelTests::tests_authenticated_dashboard_view - django.core.exceptions.FieldError: Invalid f...
FAILED dashboard/tests/tests_dashboard_views.py::QuestionModelTests::tests_unauthenticated_dashboard_view - django.core.exceptions.FieldError: Invalid...
```

--- a/shynet/core/factories.py
+++ b/shynet/core/factories.py
@@ -1,20 +1,21 @@
-from django.contrib.auth import get_user_model
 import factory
+from django.contrib.auth import get_user_model
+from factory import post_generation
 from factory.django import DjangoModelFactory
+
 from .models import Service
 
 
 class UserFactory(DjangoModelFactory):
     username = factory.Faker("user_name")
     email = factory.Faker("email")
-    name = factory.Faker("name")
+    first_name = factory.Faker("name")
 
     @post_generation
     def password(self, create, extracted, **kwargs):
         password = (
             extracted
-            if extracted
-            else factory.Faker(
+            or factory.Faker(
                 "password",
                 length=42,
                 special_chars=True,
@@ -23,6 +24,7 @@ class UserFactory(DjangoModelFactory):
                 lower_case=True,
             ).evaluate(None, None, extra={"locale": None})
         )
+
         self.set_password(password)
 
     class Meta:

